### DISCUSSION TITLE
What is Rome's version management strategy?

### DISCUSSION THREAD

**Discussion by lzm0x219** [rome/tools#3936] [Q&A] [answered]
Rome's 10.0.1 version has entered the official version, why is the nightly version still 10.0.1 instead of 10.0.2? The current approach is a bit strange, a bit unclear...

**Comment by MichaReiser** [comment, marked as answer]
Hey @lzm0x219 

We agree that nightly releases should use the next version rather than the current one. This issue is tracked in #3776 but we haven't yet had the time to re-work our CI pipeline to make this possible. But we plan to do this soon.
